internal/providers/terraform/terragruntRegistryService.go

```diff
@@ -0,0 +1,345 @@
+package terraform
```

Comment:
Should we add a comment to point to where we copied this from/give credit?
I'm also not sure if we should rename this file `terragrunt_registry_service.go`, but don't have a strong opinion.

Reply:
Oops 🙂.  I'm surpised lint didn't catch that.